src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -882,18 +877,19 @@ namespace winrt::TerminalApp::implementation
         return -1;
     }
 
-    void TerminalPage::_SetFocusedTabIndex(int tabIndex)
+    winrt::fire_and_forget TerminalPage::_SetFocusedTabIndex(int tabIndex)
     {
         // GH#1117: This is a workaround because _tabView.SelectedIndex(tabIndex)
         //          sometimes set focus to an incorrect tab after removing some tabs
         auto tab = _tabs.at(tabIndex);
```

Comment:
Should this line be after the `co_await`?

Reply:
`auto tab = _tabs.at(tabIndex);` appears before the `RunAsync` call which is why it's before the `co_await`. Let me know if there's something I'm not seeing here / rationale I'm missing.

Reply:
This was mostly a thought experiment, it seems fine in either place tbh